program_intent_engagement/apps/api/serializers.py

```diff
@@ -1,4 +1,33 @@
-# Serializers that can be shared across multiple versions of the API
-# should be created here. As the API evolves, serializers may become more
-# specific to a particular version of the API. In this case, the serializers
-# in question should be moved to versioned sub-package.
+"""
+Serializers for the program-intent-engagement API
+"""
+from rest_framework import serializers
+
+from program_intent_engagement.apps.core.models import ProgramIntent, CERTAINTY
+
+class ProgramIntentSerializer(serializers.ModelSerializer):
```

Comment:
`ModelSerializers` automatically generate serializer fields based on the Django model. You're explicitly defining serializer fields below, which defeats some of the benefits of the `ModelSerializer` over the `Serializer`. Take a look at the documentation and see what direction you want to go in: using a regular `Serializer` or a `ModelSerializer` that generates these automatically. Let me know if you need any help working through that!